src/runtime/runtime.ts

```diff
@@ -55,6 +55,35 @@ type SpawnArgs = {
 
 let runtime: Runtime | null = null;
 
+class SimpleStoreRegistry implements StoreRegistry {
+  private context: Manifest;
+
+  constructor(context) {
+    this.context = context;
+  }
+
+  // TODO: This is a temporary method to allow sharing stores with other Arcs.
+  registerStore(store: UnifiedStore, tags: string[]): void {
+    if (!this.context.findStoreById(store.id) && tags.includes('shared')) {
+      this.context['_addStore'](store, tags);
+    }
+  }
+
+  // Temporary method to allow sharing stores with other Arcs.
+  unregisterStore(storeId: string, tags: string[]) {
```

Comment:
this is somewhat overlapping with https://github.com/PolymerLabs/arcs/pull/4285 (added you as a reviewer there as well) - whichever lands first, the other needs an update.